build-script.py

```diff
@@ -170,30 +170,46 @@ def generate_gyb_files(gyb_exec, verbose, add_source_locations, destination=None
     check_gyb_exec(gyb_exec)
     check_rsync()
 
-    swiftsyntax_sources_dir = os.path.join(PACKAGE_DIR, "Sources", "SwiftSyntax")
```

Comment:
There is some duplicated code here. 
Was thinking of making a new method to avoid this? 

Should I do that?

Reply:
Would be great. I didn’t realize there would be this much code duplication.

Also: Could you make sure that the lines are at most 80 columns. It’s not a hard requirement here, but would be nice if they didn’t get too long.

Reply:
I have tried to refactor, but the `BASE_KIND_FILES` depend on the destination from SwiftSyntax, so I couldn't find a generic way. But I rested some duplicated code